cli/src/utils/initialize_joi.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+// Issues a dummy joi validation to force joi to initialize its scripts.
+// This is used because tests will mock the filesystem, and the lazy
+// `require`s done by joi will no longer work at that point.
+module.exports = (joi) =>
+  joi.validate('', joi.any().when('', { is: '', then: joi.any() }));
```

Comment:
According to @marsup, calling `Joi.options()` would also be sufficient.

https://gitter.im/hapijs/joi?at=57a1236747659bfb108977f7